scripts/centos.dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Build the test and build container for presto_cpp
+ARG image=quay.io/centos/centos:stream8
+FROM $image
+ARG cpu_target=avx
+ENV CPU_TARGET=$cpu_target
```

Comment:
We don't need CPU_TARGET as the helper function can auto infer the target. Can we remove this?

Reply:
I just carried this over from the previous file so :shrug:. I am still unclear on if/why we (still) need the  script and would like to replace it with cmake.

Reply:
Sounds good!